runner/local/simple.go

```diff
@@ -13,22 +13,34 @@ import (
 
 func NewSimpleRunner(exec execer.Execer, checkouter snapshot.Checkouter, outputCreator runner.OutputCreator) runner.Runner {
```

Comment:
As per our discussion, let's leave this as is unless we can find a solution without any drawbacks.